x-pack/platform/plugins/shared/streams/server/routes/dashboards/route.ts

```diff
@@ -158,51 +154,6 @@ const unlinkDashboardRoute = createServerRoute({
   },
 });
 
-const suggestDashboardsRoute = createServerRoute({
```

Comment:
We already have this route here: https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/streams/server/routes/internal/dashboards/route.ts